radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java

```diff
@@ -17,52 +17,203 @@
 
 package com.radixdlt.consensus;
 
-import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 import com.radixdlt.crypto.ECKeyPair;
+import com.radixdlt.crypto.ECPublicKey;
+import com.radixdlt.examples.tictactoe.Pair;
 import io.reactivex.rxjava3.core.Observable;
+import org.assertj.core.api.Condition;
+import org.junit.Test;
+
 import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Random;
+import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import org.junit.Test;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 
 /**
- * Tests with networks with crashed nodes
+ * Tests with networks with crash-stopped nodes (that do not recover).
+ * These tests comprise both static and dynamic configurations (where nodes crash before or during runtime, respectively).
  */
 public class CrashFaultNetworkTest {
-	static List<ECKeyPair> createNodes(int numNodes) {
+	static List<ECKeyPair> createNodes(int numNodes, String designation) {
```

Comment:
This isn't the most beautiful way to do this, but it's fast enough and very useful for debugging. However, it only works for static configurations (since a node short name does not change over time). We should consider adding the ability to set a custom name to BFT instances, if just for these kind of tests (e.g. to mark them as "faulty 1" or whatever).